README.md

```diff
@@ -1,21 +1,187 @@
-# cci_cameroon
+<!-- #region -->
 
-## Setup
+# Collective Crisis Intelligence Project for The Cameroon Red Cross
 
-- Meet the data science cookiecutter [requirements](http://nestauk.github.io/ds-cookiecutter/quickstart), in brief:
-  - Install: `git-crypt` and `conda`
-  - Have a Nesta AWS account configured with `awscli`
-- Run `make install` to configure the development environment:
-  - Setup the conda environment
-  - Configure pre-commit
-  - Configure metaflow to use AWS
+**_Public repository for hosting the technical outputs of the CCI Cameroon project._**
 
-## Contributor guidelines
+## Welcome!
 
-[Technical and working style guidelines](https://github.com/nestauk/ds-cookiecutter/blob/master/GUIDELINES.md)
+This repository contains the code and documentation for a project uses Collective Crisis Intellegence (CCI) with the aim to help the Cameroon Red Cross to track and respond to misinformation in realtime.
```

Comment:
Hi @georgerichardson I think the problem was because the library `textwrap` needed to be updated to `textwrap3`. Can you try running make install again after pulling the changes? Thanks.